packages/app/src/app/pages/common/Modals/SelectWorkspaceToUpgrade/index.tsx

```diff
@@ -0,0 +1,171 @@
+        </Text>
+        {teamsToShow.length > 0 ? (
+          <Select onChange={e => setSelectedTeam(e.target.value)}>
+            {teamsToShow.map(team => (
+              <option key={team.id} value={team.id}>
+                {team.name}
+              </option>
+            ))}
+          </Select>
+        ) : (
+          <Text
+            css={{
+              color: '#808080',
+              fontSize: '13px',
+              lineHeight: '19px',
+            }}
+          >
+            You don&apos;t have any teams to upgrade to Team Pro.
+            <Button
+              css={{
+                display: 'inline',
+                height: 'auto',
+                padding: 0,
+              }}
+              onClick={() => {
+                track(
+                  'subscription page - upsell team pro create team clicked',
+                  {
+                    codesandbox: 'V1',
+                    event_source: 'UI',
+                  }
+                );
+
+                openCreateTeamModal();
+              }}
+              variant="link"
+              autoWidth
+            >
+              Create one and start a free trial.
+            </Button>
+          </Text>
+        )}
+      </Stack>
+      <Stack direction="vertical" gap={2}>
+        {checkout.status === 'error' && (
+          <Text
+            css={{
+              display: 'block',
+              marginTop: '2px',
+            }}
+            variant="danger"
+            size={12}
+          >
+            An error ocurred while trying to load the checkout.
+            <br /> Please try again.
+          </Text>
+        )}
+
+        <Stack gap={2} align="center" justify="flex-end">
+          <Button
+            onClick={() => {
+              track('subscription page - upsell team pro modal closed', {
+                codesandbox: 'V1',
+                event_source: 'UI',
+              });
+              modalClosed();
+            }}
+            autoWidth
+            variant="ghost"
+          >
+            Cancel
+          </Button>
+          <Button
+            css={{
+              gap: '4px',
+              padding: '4px 24px',
+            }}
+            disabled={!selectedTeam}
+            loading={checkout.status === 'loading'}
+            variant="primary"
+            onClick={() => {
+              track('subscription page - upsell team pro checkout clicked', {
+                codesandbox: 'V1',
+                event_source: 'UI',
+              });
+
+              createCheckout({
+                team_id: selectedTeam,
+                recurring_interval: 'month',
+                success_path: '/pro',
+                cancel_path: '/pro',
+              });
+            }}
+            autoWidth
+          >
+            Checkout
+            <Element
+              css={{
+                transform: 'rotate(270deg)',
+              }}
+            >
+              <Icon name="arrowDown" size={12} />
```

Comment:
Are arrows something we're doing in buttons?

Reply:
@tristandubbeld I'm not sure, but here is the Figma for reference:

![image](https://user-images.githubusercontent.com/24959348/222155660-7b529deb-d145-4679-82ac-e1d56b48cc76.png)